src/getColumnDataType.ts

```diff
@@ -1,4 +1,7 @@
-export const getColumnDataType = (dataType: string, columnType: string): string => {
+export const getColumnDataType = (dataType: string|null, columnType: string): string => {
+  if (columnType === null) {
+    throw new Error('The DATA_TYPE field in information_schema should never be null. This may be a bug');
+  }
```

Comment:
It turns out INFORMATION_SCHEMAS says that DATA_TYPE is nullable. idk when this happens, but I guess this tool helped avoid a potential future bug ;)